.azuredevops/platformPipelines/platform.updateReadMe.yml

```
@@ -34,42 +34,48 @@ jobs:
           pwsh: true
           script: |
             # Load used functions
-            . (Join-Path '$(System.DefaultWorkingDirectory)' 'utilities' 'tools' 'Set-GitHubReadMeModuleTable.ps1')
+            . (Join-Path '$(System.DefaultWorkingDirectory)' 'utilities' 'tools' 'Set-ReadMeModuleTable.ps1')
+
+            $OrganizationURI = '$(System.CollectionUri)'
 
             $functionInput = @{
               ModulesPath    = Join-Path '$(System.DefaultWorkingDirectory)' 'arm'
               FilePath       = Join-Path '$(System.DefaultWorkingDirectory)' 'README.md'
-              Organization   = ('$(Build.Repository.Name)').split('/')[0]
-              RepositoryName = ('$(Build.Repository.Name)').split('/')[1]
+              Organization   = $OrganizationURI.Split('/')[3]
```

I'd suggest to align this code with the subsequent task where it's also using the following:

```suggestion
              Organization   = '$(System.CollectionUri)'.Split('/')[3]
```

If you agree, we should remove line 39

---

Done